pypeit/edgetrace.py

```diff
@@ -4391,17 +4520,14 @@ def get_slitid(shape, lordloc, rordloc, islit, ypos=0.5):
 
 
 # TODO: This needs to be integrated into EdgeTraceSet
-def slit_spat_pos(tslits_dict):
+def slit_spat_pos(left, right, nspec, nspat):
```

Comment:
Shouldn't this be a class method?

Reply:
It is in the new branch I'm working on that replaces `tslits_dict` with `SlitTraceSet`.  Punting this until then.